src/components/ButtonComponent/buttonComponent.types.d.ts

```diff
@@ -0,0 +1,12 @@
+import { HTMLAttributes } from 'react';
+
+/**
+ * Types for Button
+ * @param {primary} boolean - primary style of the button
+ * @param {secondary} boolean - secondary style of the button
+ */
+
+export interface IButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  primary?: boolean;
+  secondary?: boolean;
+}
```

Comment:
1. why do you need 2 flags for one thing in your interface?
2. Replace `primary` flag with `type`

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/3e113f3ad147eb58ef2301f0ddc6984a2246951c